azuredevops/internal/service/policy/resource_branchpolicy_build_validation_test.go

```diff
@@ -0,0 +1,53 @@
+// +build all policy
+// +build !exlcude_policy
```

Comment:
1. Typo here `exlcude_policy` => **`exclude_policy`**
2. There should be a build tag `resource_branchpolicy_build_validation` in the first line
3. There should be a build tag `exclude_resource_branchpolicy_build_validation` in the second line

Reply:
Like this?

What do the build tags do?

Reply:
Go build tags are a neat feature to include or exclude code during compilation. For example the tests inside this file will only be compiled and executed if one of the tags `all`, `policy` are specified and **not** `exclude_policy`.

https://www.digitalocean.com/community/tutorials/customizing-go-binaries-with-build-tags
https://dave.cheney.net/2014/09/28/using-build-to-switch-between-debug-and-release
https://mickey.dev/posts/go-build-tags-testing/